x-pack/plugins/security_solution/cypress/e2e/detection_rules/bulk_edit_rules.cy.ts

```diff
@@ -210,7 +212,9 @@ describe('Detection rules, bulk edit', () => {
       clickAddTagsMenuItem();
       waitForMixedRulesBulkEditModal(expectedNumberOfCustomRulesToBeEdited);
 
-      checkPrebuiltRulesCannotBeModified(totalNumberOfPrebuiltRules);
+      getAvailablePrepackagedRulesCount().then((availablePrepackagedRulesCount) => {
```

Comment:
inside checks for:
```typescript
    const prebuiltRulesCount = body.rules_installed + body.rules_not_installed;

```

but the number of prebuilt rules displayed in UI is the number of installed.
So, correctly would to assert this value against number of installed rules

Reply:
> but the number of prebuilt rules displayed in UI is the number of installed.
> So, correctly would to assert this value against number of installed rules

On the `loadPrebuiltDetectionRulesFromHeaderBtn()` step, all available prebuilt rules are getting installed, so it is okay to check if the UI displays all rules count.